less/common/mixins/light-contents.less

```diff
@@ -1,26 +1,26 @@
 // This is a mixin which styles components (buttons, inputs, etc.) for use on
 // dark backgrounds.
-.light-contents(@color: #fff, @control-bg: fade(#000, 10%), @control-color: #fff) {
+.light-contents(@color: #fff; @control-bg: fade(#000, 10%); @control-color: #fff; @name: 'light-content') {
```

Comment:
Shouldn't the variables args be removed here, since it should pull those from the corresponding `light-contents-vars`?

Reply:
backwards compatibility ;), if we remove the args here, any extensions using the mixin will break.

Reply:
Gotcha. Should we deprecate it?

Reply:
Well, we still use it ourselves in the code, unless you mean creating a new one with a different name and then deprecating this one.

I think it's fine to leave though, the first two args are still used if someone wants to not use CSS vars and keep it inline, unless we'd rather enforce the CSS var usage, which might be better.